nautobot/extras/tests/test_api.py

```diff
@@ -661,6 +661,28 @@ def test_run_git_sync_with_permissions(self, mock_get_worker_count):
         response = self.client.post(url, format="json", **self.header)
         self.assertHttpStatus(response, status.HTTP_200_OK)
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
+    @skipIf(
+        "dummy_plugin" not in settings.PLUGINS,
+        "dummy_plugin not in settings.PLUGINS",
+    )
```

Comment:
Is this still necessary? I thought we always load the example plugin in the CI pipeline?